### ISSUE TITLE
Understanding the identification.c File and Alternatives for System Information Retrieval

### ISSUE THREAD

**Issue by GodinhoBlaster** [FWGS/xash3d-fwgs#1630]
**Hello xash3d team,**

During my exploration of the project's source code, I came across the `identification.c` file. It seems to be responsible for obtaining system information, and I am interested in gaining a more in-depth understanding of its purpose and functionality.

1. **Purpose and Functionality:**
   - Could you provide a detailed explanation of the `identification.c` file's purpose and how it retrieves system information?

2. **Console Unavailability in C++/CX UWP:**
   - Given that I am working in a C++/CX UWP environment where the console may not be available, I am considering using the `Windows.Security.ExchangeActiveSyncProvisioning` library to obtain system information, as exemplified in the code below. Is this approach compatible with xash3d logic, or do you recommend any other alternatives?

```cpp
#include <Windows.Security.ExchangeActiveSyncProvisioning.h>
using namespace Windows::Security::ExchangeActiveSyncProvisioning;

EasClientDeviceInformation^ deviceInfo = ref new EasClientDeviceInformation();
Platform::String^ deviceManufacturer = deviceInfo->SystemManufacturer;
Platform::String^ deviceModel = deviceInfo->SystemProductName;

**Comment by nekonomicon** [comment]
> the console may not be available

IMHO will be better to read values from registry instead of current logic: 
https://stackoverflow.com/questions/21658782/c-windows-bios-serial-number-how-to-get

**Comment by a1batross** [comment]
Yep, it's just a sophisticated way to generate hardware IDs.

It's a legacy feature from the old engine. You may not even implement it, if you don't wanna to. We might rework it after we drop old protocol support.

But you still can try to find something similar in UWP APIs, and implement it in platform-specific way.

**Comment by a1batross** [comment]
I think it might be better for you, if you split UWP and Win32 target, as I can understand the set of platform APIs doesn't always overlap.

**Comment by GodinhoBlaster** [comment]
I would like to express my sincere gratitude for the technical guidance provided recently, which was crucial for the development of my project.



> I think it might be better for you, if you split UWP and Win32 target, as I can understand the set of platform APIs doesn't always overlap.

I completely agree with you. My first attempt was to keep the Win32 platform, believing it would make the job easier. However, I only encountered problems, which led me to remove it completely. I then decided to create a new UWP XBOX platform following the documentation guidelines. I chose to use SDL in conjunction, as I am aware of its excellent integration with the UWP environment. One of the biggest challenges I face is interoperability between C++/CX and C. Frequent errors arise due to incorrect includes, but I believe I am on the right track.
